pages.es/common/tmux.md

```diff
@@ -1,36 +1,37 @@
 # tmux
 
-> Multiplexa varias consolas virtuales.
+> Multiplexor de terminal. Permite múltiples sesiones con ventanas, paneles y más.
```

Comment:
```suggestion
> Multiplexor de terminal.
> Permite múltiples sesiones con ventanas, paneles y más.
```

Reply:
@gutjuri Just now checked the English page, seems like we do the same there:

```
> Terminal multiplexer. It allows multiple sessions with windows, panes, and more.
> See also `zellij` and `screen`.
> More information: <https://github.com/tmux/tmux>.
```

That's because we generally suggest having 3 lines of information only (with very rare exceptions). IG we can leave it as it is. If you prefer having it as separate lines then we need to update the English page and translations of `tmux` to follow the same.